src/commands/write.ts

```diff
@@ -97,12 +97,6 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
         normalized.statements.map(async function (stmt) {
           // re-normalize so we can be sure we've isolated each statement and it's tableId
           const norm = await normalize(stmt);
-          /* c8 ignore next 5 */
-          if (norm.tables.length > 1) {
-            throw new Error(
-              "cannot normalize if single query affects more then one table"
-            );
-          }
           const { tableId } = await globalThis.sqlparser.validateTableName(
```

Comment:
Before this change, the array returned from `normalize()` consisted of only a single item at this point in execution. Now, there could be statements where there first tableId return from normalize may not be the tableId being mutated. That seems wrong, somehow. 

This might be moot though, I don't see the tableId being used later, but it seems like it could trip someone up later.

Reply:
hmm yeah, the regex logic i had to add to the SDK had to deal with this—i.e., the `tableId`s returned for a INSERT + subselect included both table IDs, even though there was only one table being mutated. feels like the sqlparser should handle these types of things?

Reply:
e.g., i feel like the parser should do the regex-type logic and return `tables: { read: [...], write: [...] }` (unrelated: sorta like how the aliases config file sets things up, in a way)